src/wp-includes/ms-files.php

```diff
@@ -8,6 +8,7 @@
  * @subpackage Multisite
  */
 
+define( 'MS_FILES_REQUEST', true );
```

Comment:
I am OK with this approach, and here's why...

Doing some research, there are (obviously) other `_REQUEST` constants:
* `IFRAME`
* `XMLRPC`
* `REST`

If `IFRAME_REQUEST` didn't already exist, I'd probably argue that these constants were reserved for external APIs and `MS_FILES` shouldn't use that pattern.

The existence of `IFRAME_REQUEST` makes adding `MS_FILES_REQUEST` fine to do.

It is a unique type of request that requires special consideration and handling.